sources/packages/backend/libs/test-utils/src/factories/disbursement-schedule.ts

```diff
@@ -34,3 +34,31 @@ export function createFakeDisbursementSchedule(relations?: {
   schedule.disbursementScheduleStatus = DisbursementScheduleStatus.Pending;
   return schedule;
 }
+
+/**
+ * Disbursement awards.
+ */
+interface DisbursementAwards {
+  [disbursementValueCode: string]: number;
+}
```

Comment:
Not sure, if we are adding the interface in the same file in `api/`, if we are then pls move it to the top or move it to the respective file